### PR TITLE
convolutional.py create  SpatialActivation2D, add to DenseNetFCN

### PR DESCRIPTION
**Pull request by ahundt**
Compute the spatial softmax or another activation of a convolutional feature map.
        Based on the paper *Learning visual feature spaces for robotic manipulation with
        deep spatial autoencoders.* Finn et. al, http://arxiv.org/abs/1509.06113.

### CODE REVIEW

**Comment by ahundt** [comment]
Oh also this is based on the implementation of spatial_softmax in master, but that isn't in any release yet https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/layers/python/layers/layers.py#L2619

**Comment by gabrieldemarmiesse** [comment]
@ahundt The code seems fine but we have new guidelines for contributions. If you still want this merged, could you please open a new PR for layer and a new PR for the model? With tests and yourself added to the codeowners? Thank you.
